src/kbmod_wf/resource_configs/klone_configuration.py

```diff
@@ -62,18 +63,19 @@ def klone_resource_config():
                 ),
             ),
             HighThroughputExecutor(
+                # This executor is used for reprojecting sharded WorkUnits
                 label="sharded_reproject",
                 max_workers=1,
                 provider=SlurmProvider(
                     partition="ckpt-g2",
                     account="astro",
                     min_blocks=0,
-                    max_blocks=2,
+                    max_blocks=10,
                     init_blocks=0,
                     parallelism=1,
                     nodes_per_block=1,
-                    cores_per_node=32,
-                    mem_per_node=128,  # ~2-4 GB per core
+                    cores_per_node=8,
+                    mem_per_node=32,  # ~2-4 GB per core
```

Comment:
In this executor we're cranking up the maximum number of concurrent jobs running and decreasing the cores per node and memory.